src/Chat.test.tsx

```diff
@@ -3,6 +3,45 @@ import { render } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
 import { Chat } from "./Chat-after-refactoring";
+import { Message, Messagerie, Subscribe, Unsubscribe } from "./Model";
+
+/*
+ * Implémentation de test naïve, en mémoire de notre module de messagerie.
+ */
+class MessagerieTestImplementation implements Messagerie {
```

Comment:
avec MIWE, on aime bien appeler `MessagerieTestImplementation` => `InMemoryMessagerie`

Reply:
Oui je fais aussi ça sur mes projets, mais dans le cadre de l'article tu trouves pas que c'est plus explicite de dire que c'est une implémentation de test ?

Reply:
Ouais why not.. J'avoue que je sais pas trop lol